client/src/app/pages/dashboard/maps/map-list/map-list.component.html

```diff
@@ -32,6 +56,9 @@ <h3 class="m-0" *ngSwitchCase="mapListType.TYPE_UPLOADS">
       <i>You have not uploaded any maps. Try <a routerLink="/dashboard/maps/uploads/new">uploading a new one!</a></i></h3>
   </div>
 </div>
+<div class="text-center" *ngIf="maps.length === 0 && requestSent && isSearchFiltered()">
+  <h3 class="m-0">{{getNoMapsFoundText()}}</h3>
```

Comment:
What I meant here was to assign it to a local variable. Since bindings like this will be evaluated multiple times, this method would still be called when it should ideally know what type it is on `ngInit`, where a single string could be assigned and then here could reference it in interpolation.